apps/gateway/tsup.config.ts

```diff
@@ -23,5 +23,9 @@ export default defineConfig({
     '.html': 'empty',
   },
   external: [...rootDependencies, ...dependencies],
-  noExternal: ['@latitude-data/env', '@latitude-data/core'],
+  noExternal: [
+    '@latitude-data/env',
+    '@latitude-data/core',
+    '@latitude-data/constants',
```

Comment:
I added `@latitude-data/constants` to all our apps. This package would be the common package without other dependencies where we put all the shared constants and types.